vignettes/bds_finding.Rmd

```diff
@@ -611,7 +766,7 @@ ad_advs.R | Derive a new PARAMCD based on existing PARAMCDs (MAP, based on Systo
 Example 1 (Creating a new record):
```

Comment:
Just wondering if we need the sample code/description if we list the example below?

Reply:
Removed. Done.